.github/workflows/sbt.yml

```diff
@@ -69,9 +69,9 @@ jobs:
           - 8
         spark:
           - '2.4'
-        scala-binary:
```

Comment:
Does all `scala-binary` have no usage?

Reply:
Yea, I think it's safe to remove.

Reply:
I did some validation tests, since in sbt we support multiple scala version tests, it makes sense to use `scala-binary`.

Reply:
@cxzl25, as you said, the scala-binary of Spark 2.4 shoud also not be removed.